Cargo.toml

```diff
@@ -52,7 +52,7 @@ env_logger = "0.10"
 eyre = "0.6.12"
 itertools = "0.12"
 jsonrpsee = { version = "0.21.0", features = ["client", "macros", "tokio", "server"] }
-rand = "0.8"
+rand = { version = "0.8", features = ["small_rng"] }
```

Comment:
Looks like it needed only in the dev-dependencies.

Reply:
I cant quickly find a way to put it into `workspace.dev-dependencies` and then inherit it.

Reply:
Can you define specific flags in specific crate's dev-dependencies?  It is OK to define specific flags, workspace is essentially needed for versioning.